Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -3186,7 +3187,7 @@ static WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     Vector<PageIdentifierTuple> result;
 
     for (auto entry : m_popupPageActionMap) {
-        RefPtr tab = entry.value->tab();
+        RefPtr tab = Ref { entry.value }->tab();
```

Comment:
Use a local var.

Reply:
Actually, (feel free to correct me if I'm wrong), but I believe `entry.value` is a Ref on the stack here (since the for-each loop should be by copy here, so the Ref is created right there). So we don't need to Ref it here. I just removed my modification in the newest iteration of the patch.